client/components/web-preview/toolbar.jsx

```diff
@@ -93,7 +104,7 @@ class PreviewToolbar extends Component {
 						aria-label={ translate( 'Close preview' ) }
 						className="web-preview__close"
 						data-tip-target="web-preview__close"
-						onClick={ onClose }
+						onClick={ onClose } // **HERE**
```

Comment:
Anyone know how to add a Tracks event here? I tried adding my own function that calls both `onClose ` and `analytics.tracks.recordEvent`, but no dice. It records the Tracks event but does not close the preview.